fauna/manifest.json

```diff
@@ -0,0 +1,53 @@
+{
+  "manifest_version": "2.0.0",
+  "app_uuid": "2be7cc0c-a21f-43ad-b2b7-3f41a98ad299",
+  "app_id": "fauna",
+  "display_on_public_website": true,
+  "tile": {
+    "overview": "README.md#Overview",
+    "configuration": "README.md#Setup",
+    "support": "README.md#Support",
+    "uninstallation": "README.md#Uninstallation",
+    "changelog": "CHANGELOG.md",
+    "description": "Import your Fauna query logs into Datadog.",
+    "title": "Fauna",
+    "media": [
+      {
+        "media_type": "image",
+        "caption": "Fauna Dashboard Metrics",
+        "image_url": "images/fauna_dashboard_1.png"
+      },
+      {
+        "media_type": "image",
+        "caption": "Fauna Log Example",
+        "image_url": "images/fauna_log_example.png"
+      }
+    ],
+    "classifier_tags": [
+      "Supported OS::Linux",
+      "Supported OS::Windows",
+      "Supported OS::macOS",
+      "Category::Cloud",
+      "Category::Data Stores",
+      "Category::Log Collection",
+      "Offering::Integration",
+      "Submitted Data Type::Logs"
+    ]
+  },
+  "assets": {
+    "oauth": "assets/oauth_clients.json",
+    "integration": {
```

Comment:
what should I be putting for `source_type_id`? I see in the docs that it was generated when creating the integration scaffolding, I must have missed recording this at the time. Is there a way to obtain or regenerate this value?

Reply:
I generated a source_type_id for you and included it in a suggestion. This should fix the failing validations.